src/components/__tests__/SortPicker.ts

```diff
@@ -380,3 +380,127 @@ describe('Created to Updated', () => {
     expect(thoughtValues).toMatchObject(['a', 'c', 'd'])
   })
 })
+
+it('home: Note Asc', async () => {
+  act(() => {
+    store.dispatch([
+      importText({
+        text: `
+          - a
+            - =note
+              - 2
+          - b
+            - =note
+              - 3
+          - c
+            - =note
+              - 1
+        `,
+      }),
+      setCursor(['a']),
+    ])
+  })
+
+  await act(vi.runOnlyPendingTimersAsync)
+
+  await click('[data-testid="toolbar-icon"][aria-label="SortPicker"]')
+  await click('[aria-label="sort options"] [aria-label="Note"]')
+  await act(() => vi.runAllTimersAsync())
+
+  const thoughts = screen.getAllByTestId(/thought/)
+  expect(thoughts.map((child: HTMLElement) => child.textContent)).toMatchObject(['c1', 'a2', 'b3'])
+})
+
+it('home: Note Desc', async () => {
+  act(() => {
+    store.dispatch([
+      importText({
+        text: `
+          - =sort
+            - Note
+          - a
+            - =note
+              - 2
+          - b
+            - =note
+              - 3
+          - c
+            - =note
+              - 1
+        `,
+      }),
+      setCursor(['a']),
+    ])
+  })
+
+  await act(vi.runOnlyPendingTimersAsync)
+
+  await click('[data-testid="toolbar-icon"][aria-label="SortPicker"]')
+  await click('[aria-label="sort options"] [aria-label="Note"]')
+  await act(() => vi.runAllTimersAsync())
+
+  const thoughts = screen.getAllByTestId(/thought/)
+  expect(thoughts.map((child: HTMLElement) => child.textContent)).toMatchObject(['b3', 'a2', 'c1'])
+})
+
+it('home: Note Asc', async () => {
```

Comment:
This test looks identical to the one above. Did you have something else in mind?